src/rules/selector-pseudo-class-parentheses-space-inside/README.md

```diff
@@ -0,0 +1,53 @@
+# selector-pseudo-class-parentheses-space-inside
+
+Require a single space or disallow whitespace on the inside of the parentheses within selectors.
```

Comment:
"Require a single space or disallow whitespace on the inside of the parentheses within pseudo-class selectors."